### ISSUE TITLE
No sensorvalue from flowsensor

### ISSUE THREAD

**Issue by ChristianNiklasson** [radical-squared/aquatemp#85]
In the older version of the integration i was able to get a fault message "no flow" or similar.
That message is no longer available and a state that i would like to have since i can activate my chemical equipment based on that entity/state

**Comment by elad-bar** [comment]
you should have fault sensor:
![image](https://github.com/radical-squared/aquatemp/assets/3207137/f84ae28c-474e-41fe-8450-770c934a34a8)

pls let me know if you don't find it
thanks

**Comment by ChristianNiklasson** [comment]
Yes i have it, the problem is that the state never changes, if i shut down the circulation ie the status is still "Ok"

**Comment by elad-bar** [comment]
can you pls assist me with understanding the issue? i will need data when it happens, meaning when you expect the fault to be not ok, but something else, reload the integration (from devices & services), wait few seconds for the integration to load and then extarct diagsnotic file and share it?

according to the logic in code as well as verifying it within Postman, once there is an error it should work, unless the api changed ?(for that i will need the diagnostic file).

thanks

**Comment by tgasslander** [comment]
I have the same issue, and I'd like try to help. 
I'm attaching the diagnostics file. This is when the circulation is off and the heat pump should report 'no flow' but it doesn't. I've retracted the identity of myself and the device. Hope it still has value.

[config_entry-aqua_temp-fa3b1b325c6d552bca4f0878659b93bf.json](https://github.com/user-attachments/files/16382705/config_entry-aqua_temp-fa3b1b325c6d552bca4f0878659b93bf.json)

**Comment by elad-bar** [comment]
According to the diagnostics file, is_fault = false,
This is coming from the API, means no error

**Comment by tgasslander** [comment]
Alright. Thanks.
Sadly I don't have time to investigate this further at this point.

**Comment by ChristianNiklasson** [comment]
Any progress? I would really like to get hold of this entity so I could build an automation that turns the pool pump of if there is no water flow detected :)

**Comment by molcam** [comment]
> Any progress? I would really like to get hold of this entity so I could build an automation that turns the pool pump of if there is no water flow detected :)

Yes, this feature would be great. If there is no flow through the pump, wait some seconds and then automate a stop of the circulation pump.

**Comment by majdzik84** [comment]
I'm joining the request for a repair. The fault sensor always shows OK, even when there's no flow.

**Comment by alistairts** [comment]
Any progress with this fix. Fault sensor is pointless if its state remains unchanged.
